yaptide/routes/user_routes.py

```diff
@@ -72,13 +74,40 @@ def get(user: UserModel):
                         'input_type': simulation.input_type,
                         'sim_type': simulation.sim_type
                     }
-                }
-                for simulation in simulations],
-            'page_count': pagination.pages,
-            'simulations_count': pagination.total,
+                } for simulation in simulations
+            ],
+            'page_count':
+            pagination.pages,
+            'simulations_count':
+            pagination.total,
         }
         return yaptide_response(message='User Simulations', code=200, content=result)
 
+    @staticmethod
+    @requires_auth()
+    def delete(user: UserModel):
+        """Method deleting simulation from database"""
+        schema = UserSimulations.APIParametersSchema()
```

Comment:
DELETE requires only `job_id`. It would be better to define its own marchmallow schema, without `order_by` and other stuff needed by GET.

```
class DeleteAPIParametersSchema(Schema):
    """Schema for DELETE method parameters"""
    job_id = fields.String(required=True)  # job_id is mandatory for DELETE
```